pennylane_lightning/core/src/simulators/lightning_gpu/measurements/MeasurementsGPU.hpp

```diff
@@ -135,6 +135,7 @@ class Measurements final
             /* const int32_t* */ maskBitString,
             /* const int32_t* */ maskOrdering,
             /* const uint32_t */ maskLen));
+        PL_CUDA_IS_SUCCESS(cudaStreamSynchronize(this->_statevector.getDataBuffer().getDevTag().getStreamID()));
```

Comment:
hmmm, yea, `_statevector` is a private data member declared in the `MeasurementBase` class. Then all good!

Reply:
Isn't it protected, rather than private?

Reply:
yea, it's protected.